code/game/objects/items/devices/organ_module/active/hud_types.dm

```diff
@@ -4,39 +4,44 @@
 	verb_name = "Activate Med Hud"
 	icon_state = "medshades"
 	holding_type = /obj/item/clothing/glasses/hud/health/shades
-	price_tag = 575
+	matter = list(MATERIAL_STEEL = 12)
```

Comment:
put these values on /hud/ if possible, then you can cutdown on most of the copypaste here